### DISCUSSION TITLE
Items and errors?

### DISCUSSION THREAD

**Discussion by ImMorphi** [Povstalec/StargateJourney#102] [Q&A] [answered]
I am trying to find the items that show in JEI within this mod (player shield emitter specifically). I have been mainly using creative + structure compass and locate commands to find the stargates and teleporting to the different dimensions. Looking in the chests that generate loot usually will have just a few of the sgjourney materials but searching over 20 abydos pyramids, I havent found any. I personally have never watched the show so not entirely sure what im supposed to be looking for each.

Also I don't know if this is common or not but I was trying to go to Lantea, and was successful. Portaled to it went underwater swam for a bit, game crashed. Reloaded, was put back in previous dimension, put portal code back in for Lantea "No stargates on this dialed planet" something along those lines which is weird and im not sure if its corrupted or if im doing something wrong?

Playing ATM9 v0.2.56, with sgjourney v0.6.22.

Cheers for any help :D

**Comment by lukaskabc** [comment, marked as answer]
> I am trying to find the items that show in JEI within this mod (player shield emitter specifically). I have been mainly using creative + structure compass and locate commands to find the stargates and teleporting to the different dimensions. Looking in the chests that generate loot usually will have just a few of the sgjourney materials but searching over 20 abydos pyramids, I havent found any. I personally have never watched the show so not entirely sure what im supposed to be looking for each.  

They are not obtainable in survival, much of the content is WIP and not yet implemented.

> Also I don't know if this is common or not but I was trying to go to Lantea, and was successful. Portaled to it went underwater swam for a bit, game crashed. Reloaded, was put back in previous dimension, put portal code back in for Lantea "No stargates on this dialed planet" something along those lines which is weird and im not sure if its corrupted or if im doing something wrong?

Do you have crash report and log from that session?

**Reply by ImMorphi** [reply]
I unfortunately do not have the crash log from what I can see. In the Lantea dimension, i used the structure compass to find the outpost, it told me it was way under the sea, so I started diving and about half way through the game started stuttering every 1-3 seconds. Task Manager showed that my CPU was spiking every stutter but wasn't spiking to 100, just from 70%->80%. Sorry I can't be much more help than that, there is a good chance its unrelated to this mod and just ATM9 issues. 

In regards to the items, it is what I suspected however in that case I am very excited for the future of this mod as I really like how the player shield emitter works, bouncing back enemies around you so that they can't attack you / negating damage. ZPM device sounds scary though earth shattering device might be dangerous in the wrong hands.

Thank you for the rapid response!

**Reply by Povstalec** [reply]
I'll most likely make the Personal Shield Emitter craftable once I make it powered by Liquid Naquadah instead of regular energy (for balancing reasons related to being able to turn off energy for the entire mod, which would mean the shield would last forever without recharging) and add the Stargate Iris/Shield. Eventually, of course, I do want it to be something you can find in loot in Ancient structures too.

As for the crash, I don't know, it oculd be simply that there was too much to load at once, since due to its very deep ocean, Lantea's chunks have a height of 512, as opposed to every other planet's 256, which makes generation more resource intense I suppose.
